### DISCUSSION TITLE
NEED HELP / GitHub Pages Site Not Updating and Now Returning 404 Error

### DISCUSSION THREAD

**Discussion by KageRyo** [community/community#110636] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello everyone,
I'm encountering a critical issue with my GitHub Pages site, and I'm seeking urgent assistance from the community to resolve it.
Website: https://kageryo.coderyo.com/
Repo: https://github.com/KageRyo/MyWebsite

### Problem Description
1. Previously, my GitHub Pages site was functioning correctly, displaying the latest updates from my repository.
2. However, recently, it stopped updating automatically. 
3. After attempting to adjust the settings to rectify this issue, my site is now returning a 404 error, indicating that it cannot be found.

### What I've Tried
- Checked GitHub Pages settings to ensure correct branch and folder path.
- Verified the repository's status and branch integrity.
- Ensured the file structure and content in the repository are correct.
- Cleared browser cache and tried accessing the site from different browsers.

### Request for Assistance
I urgently need assistance in resolving this issue. If anyone has experienced a similar problem or has any insights into what might be causing this sudden change in behavior, your guidance would be greatly appreciated.

Thank you all for your time and support.

Best regards, Chien-Hsun, Chang(KageRyo)

**Comment by AidenJamesYt** [comment, marked as answer]
Github is currently monitoring a recent incident.

You can view the status of it [here](https://www.githubstatus.com/)

it might be impacting your performance

**Reply by KageRyo** [reply]
Thank you for your response and for bringing the recent incident to my attention. I appreciate your help and understanding.

I will continue to monitor the situation.

Thanks again for your assistance.

**Reply by AidenJamesYt** [reply]
No problem! If there is anything else I can help you with, lmk! Also, if i have answered your question, please remember to mark as answer :)
